Gemfile

```diff
@@ -6,7 +6,7 @@ ruby RUBY_VERSION
 
 gemspec
 
-gem 'grape', case version = ENV['GRAPE_VERSION'] || '~> 1.0'
+gem 'grape', case version = ENV['GRAPE_VERSION'] || '~> 1.2'
```

Comment:
Leave this at 1.0 for now, but add another entry to .travis.yml with the branch, so for example

```ruby
when 'INSTANCE'
 { git: '...', branch: '...' } # your own PR branch here
when 'HEAD'
  ...
```